examples/webgl_materials_cubemap_mipmaps.html

```diff
@@ -28,21 +28,24 @@
 			init();
 			animate();
 
-			//load custmized cube texture
+			//load customized cube texture
 			async function loadCubeTextureWithMipmaps() {
 
 				var path = 'textures/cube/angus/';
 				var format = '.jpg';
 				var mipmaps = [];
+				var maxLevel = 8;
 
 				async function loadCubeTexture( urls ) {
 
 					return new Promise( function ( resolve, reject) {
 
-						new THREE.CubeTextureLoader().load( urls, function ( cubeTexture ) {
+						new THREE.CubeTextureLoader().load( urls,function ( cubeTexture ) {
 
 							resolve( cubeTexture );
 
+						}, function ( error ) {
```

Comment:
Any reasons for adding an `onError` callback with no implementation?

Reply:
add for debug, forget to remove  : )